core/src/main/java/org/infinispan/container/versioning/irac/DefaultIracTombstoneManager.java

```diff
@@ -117,73 +151,104 @@ public int size() {
       return tombstoneMap.size();
    }
 
+   @Override
+   public void sendStateTo(Address origin, IntSet segments) {
```

Comment:
> TBH the method name also sounds too generic, since it only sends tombstone state.

`IracTombstoneManager#sendStateTo()` <- what kind of state have you thought about? 

yeah, I'll change the `origin` parameter's name.

Reply:
Sorry, I wanted to copy your test in order to reply and I updated your comment instead :)